targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F437xG/TARGET_UBLOX_C030/min_battery_voltage.h

```diff
@@ -0,0 +1,21 @@
+#ifndef MIN_BATTERY_VOLTAGE_H
+#define MIN_BATTERY_VOLTAGE_H
+
+#ifdef __cplusplus
+extern"C"{
+#endif
+
+#define BQ24295_I2C_ADDRESS (0x6B << 1)
+#define MIN_BATTERY_VOLTAGE_MASK (0x87)
+
+/** Initializes an instance of class BatteryChargerI2c which is using the STM HAL I2C APIs
```

Comment:
Maybe add to this, just to be completely clear:

`This allows longer USB cables or USB ports with lower output voltages to power the board correctly.`